verification/cbmc_stubs/threads.c

```
@@ -7,7 +7,7 @@
 #define __CPROVER_THREADS_H_INCLUDED
 #endif
 
-int thrd_create(thrd_t *thr, thrd_start_t func, void *arg)
+int _avocado_thrd_create(thrd_t *thr, thrd_start_t func, void *arg)
```

Clang (14.0.6)

[warning] declaration uses identifier '_avocado_thrd_create', which is reserved in the global namespace

(bugprone-reserved-identifier)